ui/eureka/eureka.js

```diff
@@ -1976,181 +2185,271 @@ const Options = {
         },
         demonic: {
           label: {
-            en: 'Demonic',
+            en: 'Demonstrably Demonic',
             de: 'Der Geruch der Angst',
             fr: 'Par l\'hémoglobine alléchés',
             ja: '血の匂いに誘われて',
             cn: '闻血而来',
             ko: '피비린내에 이끌려',
           },
+          dropsFieldNotes: false,
+          shortLabel: {
+            en: 'Demonic',
+            fr: 'Hémoglobine',
+          },
           x: 11.1,
           y: 20.2,
           fateID: 1624,
         },
         absentfriends: {
           label: {
-            en: 'Absent',
+            en: 'For Absent Friends',
             de: 'Eine neue Unordnung',
             fr: 'Miséricorde vengeresse',
             ja: '燃え上がる南方戦線',
             cn: '南方战线的激战',
             ko: '타오르는 남부 전선',
           },
+          dropsFieldNotes: true,
+          fieldNotes: 12,
+          shortLabel: {
+            en: 'Absent',
+            fr: 'Vengeresse',
+          },
+          isCEPrecursor: true,
           x: 13.8,
           y: 18.3,
           fateID: 1625,
         },
         steelflame: {
           label: {
-            en: 'Steel',
+            en: 'Of Steel and Flame',
             de: 'Auf und ab',
             fr: 'Le fer et le feu',
             ja: '続・燃え上がる南方戦線',
             cn: '南方战线的续战',
             ko: '타오르는 남부 전선 속편',
           },
+          isCEPrecursor: true,
+          dropsFieldNotes: true,
+          fieldNotes: 14,
+          shortLabel: {
+            en: 'Steel',
+            fr: 'Fer & Feu',
+          },
           x: 13.8,
           y: 18.3,
           fateID: 1626,
         },
         dogsofwar: {
           label: {
-            en: 'Dogs',
+            en: 'Let Slip the Dogs of War',
             de: 'Vor die Hunde gekommen',
             fr: 'Brigade canine',
             ja: '戦場の犬を解き放て',
             cn: '释放战场之犬',
             ko: '전장에 개를 풀어라',
           },
+          dropsFieldNotes: true,
+          fieldNotes: 4,
+          shortLabel: {
+            en: 'Dogs',
+            fr: 'Brigade',
+          },
           x: 14.0,
           y: 15.3,
           fateID: 1627,
         },
         warmachines: {
           label: {
-            en: 'War',
+            en: 'The War Against the Machine',
             de: 'Ende Gelände',
             fr: 'Cent mille guerriers de métal',
             ja: 'シシニアスの実験場',
             cn: '西西尼乌斯的实验场',
             ko: '시시니우스의 실험장',
           },
+          dropsFieldNotes: true,
+          fieldNotes: 3,
+          shortLabel: {
+            en: 'War',
+            fr: 'Cent Mille',
+          },
           x: 11.1,
           y: 20.2,
           fateID: 1628,
         },
         castrumlacuslitore: {
           label: {
+            en: 'Castrum Lacus Litore',
+            de: 'Castrum Lacus Litore',
+            fr: 'Castrum Lacus Litore',
+            ja: 'カストルム',
+            cn: '湖岸堡',
+            ko: '공성전',
+          },
+          dropsFieldNotes: false,
```

Comment:
I definitely agree with not listing field notes for Castrum and Dalriada, as that might get out of hand.

Reply:
I was hesitating about this but it felt out of scope